<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EventPlannerClass</class>
 <widget class="QMainWindow" name="EventPlannerClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>599</width>
    <height>577</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>EventPlanner</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <widget class="QLabel" name="personLocation">
      <property name="minimumSize">
       <size>
        <width>161</width>
        <height>16</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>161</width>
        <height>16</height>
       </size>
      </property>
      <property name="text">
       <string>Location</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_8">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QCheckBox" name="checkBox">
            <property name="text">
             <string>Show near events</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QListWidget" name="eventsList"/>
          </item>
          <item>
           <widget class="QPushButton" name="goingButton">
            <property name="text">
             <string>Going</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QLabel" name="partyDescriptionLabel">
          <property name="text">
           <string>PartyDescription</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QListWidget" name="listOfInvited"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_7">
        <item>
         <widget class="QLabel" name="nameLabel">
          <property name="text">
           <string>Name</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="inputName"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="QLabel" name="descLabe">
          <property name="text">
           <string>Description</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="inputDescription"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QLabel" name="latLabel">
          <property name="text">
           <string>Latitude</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="inputLatitude"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="longLabel">
          <property name="text">
           <string>Longitude</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="inputLongitude"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="dLabel">
          <property name="text">
           <string>Day</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="inputDay"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="mLabel">
          <property name="text">
           <string>Month</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="inputMonth"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="yLabel">
          <property name="text">
           <string>Year</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="inputYear"/>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="addEventButton">
        <property name="text">
         <string>Add Event</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>599</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="EventPlanner.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>goingButton</sender>
   <signal>clicked()</signal>
   <receiver>EventPlannerClass</receiver>
   <slot>goingEvent()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>51</x>
     <y>242</y>
    </hint>
    <hint type="destinationlabel">
     <x>-27</x>
     <y>246</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>eventsList</sender>
   <signal>clicked(QModelIndex)</signal>
   <receiver>EventPlannerClass</receiver>
   <slot>listClickEvent()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>50</x>
     <y>112</y>
    </hint>
    <hint type="destinationlabel">
     <x>-34</x>
     <y>117</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBox</sender>
   <signal>stateChanged(int)</signal>
   <receiver>EventPlannerClass</receiver>
   <slot>checkBoxList()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>21</x>
     <y>57</y>
    </hint>
    <hint type="destinationlabel">
     <x>-17</x>
     <y>55</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>addEventButton</sender>
   <signal>clicked()</signal>
   <receiver>EventPlannerClass</receiver>
   <slot>addEvent()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>479</x>
     <y>513</y>
    </hint>
    <hint type="destinationlabel">
     <x>670</x>
     <y>512</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>goingEvent()</slot>
  <slot>listClickEvent()</slot>
  <slot>checkBoxList()</slot>
  <slot>addEvent()</slot>
 </slots>
</ui>
